webknossos/webknossos/dataset/layer.py

```diff
@@ -451,6 +475,7 @@ def add_mag_for_existing_files(
                     if mag_array_info.data_format == DataFormat.WKW
                     else None
                 ),
+                # TO DO: I think nd support is missing here.
```

Comment:
I just noticed this while reading the code. Should I open an issue for this? Or do you think this is irrelevant?